src/features/game/lib/crafting.ts

```
@@ -224,6 +256,30 @@ export const RECIPES: Recipes = {
+      { collectible: "Merino Wool" },
+      { collectible: "Merino Wool" },
+      { collectible: "Merino Wool" },
+    ],
+    time: 8 * 60 * 60 * 1000,
+    type: "collectible",
+  },
+};
+
+// Maps craftable items to the chapter they are available in.
+// Items in this config are only shown in the crafting box UI during that chapter.
+export const CHAPTER_CRAFTING_ITEMS: Partial<Record<RecipeCollectibleName, ChapterName>> = {
```

ESLint

[error] Replace `Record<RecipeCollectibleName,·ChapterName>` with `⏎··Record<RecipeCollectibleName,·ChapterName>⏎`

(prettier/prettier)